cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -322,23 +371,22 @@ void run_mnist(const std::string& model_name,
 }
 
 int main(int argc, char* argv[]) {
-  std::string context("Inference Scaling Tests");
   MPI_Init(&argc, &argv);
 
-  log_data(context, LLDebug, "Starting Inference tests");
   double main_start = MPI_Wtime();
 
   int rank;
   MPI_Comm_rank(MPI_COMM_WORLD, &rank);
-  log_data(context, LLDebug, "rank: " + std::to_string(rank) + " initiated");
+  std::string context("Inference Tests Rank " + std::to_string(rank));
+  log_data(context, LLDebug, "Rank Initialized");
 
   //Open Timing file
   std::ofstream timing_file;
   timing_file.open("rank_"+std::to_string(rank)+"_timing.csv");
 
   // Run timing tests
   run_mnist("resnet_model", "resnet_script", timing_file);
-  if(rank==0)
+  if(rank==0) //I might move this to after MPI_Wtime like in data agg
```

Comment:
We should be consistent across all tests

Reply:
I was incorrect with this